modules/sandboxed-containers-metrics-list.adoc

```diff
@@ -0,0 +1,16 @@
+// Module is included in the following assemblies:
+//
+// * sandboxed_containers/deploying_sandboxed_containers.adoc
+
+[id="sandboxed-containers-metrics-list_{context}"]
+= About {sandboxed-containers-first} metrics
```

Comment:
How does this section relate to the metrics we really exposed in the dashboard? 

So after I read this, I would expect I can see a dashboard for seeing those metrics, this does not seem to be the case.

Reply:
Good point @zanetworker : IMHO the metrics consumption is mainly via Prometheus, so can be queried via Prometheus itself.
The dashboard we provide is a simple (and sample!) one using few of all the metrics available, just showing the number of sandboxed containers workloads (kata sandboxes) running in the cluster, the CPU usage and Memory consumption for each VM.

Reply:
@zanetworker - I updated the text:

Sandboxed containers metrics enable administrators to monitor how their sandboxed containers are running. You can query for these metrics in **Metrics UI** in the web console.

Is that clearer?